Entities/User.java

```diff
@@ -0,0 +1,44 @@
+package io.github.djunicode.djcomps.Entities;
+
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.ForeignKey;
+import android.arch.persistence.room.PrimaryKey;
+
+import java.math.BigInteger;
+
+import static android.arch.persistence.room.ForeignKey.CASCADE;
+
+/**
+ * Created by Jai on 30-12-2017.
+ */
+
+@Entity(foreignKeys = @ForeignKey(entity = Group.class,
+        parentColumns = {"group_id", "student", "year", "division"},
+        childColumns = {"group_id", "student", "year", "division"},
+        onDelete = CASCADE))
+
+
+public class User {
+    @PrimaryKey public BigInteger sapid;
+    public char password;
+    public char bio;
+    public char name;
+    public float diskUtilization;
+    public BigInteger group_id;
+    public boolean student;

```

Comment:
student, year, division is already present in Group we don't need to add it again to User